src/components/Tabs.tsx

```diff
@@ -61,12 +58,12 @@ export default class Tabs extends React.Component<TabsProps, TabsState> {
     items.map((item, idx) => {
       let [name, data] = item;
       let current = idx === this.state.tab;
-      let navItem: JSX.Element = (
+      let navItem = (
         <li key={name} role="presentation" className={`tab-nav ${current ? "current" : ""}`}>
           <button
```

Comment:
Do you want to use the `Button` component here?

Reply:
In general: yes, definitely.  But I think that's a different ticket.